File: ADO/suredata2/Backup1/suredata1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Data.SqlClient;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace suredata1
{
    public partial class Form1 : Form
    {
        
        SqlConnection c;
        SqlCommand cmd;
        SqlDataReader dr;
        public Form1()
        {
            c = new SqlConnection("Data Source=SUNETHRA\\ARASU;Initial Catalog=ado;Integrated Security=True");
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            c.Open();
            cmd = new SqlCommand("insert into emp(eid,ename,salary) values(" + textBox1.Text + ",'" + textBox2.Text + "'," + textBox3.Text + ")", c);

            int a = cmd.ExecuteNonQuery();
            if (a == 0)
                MessageBox.Show("Data Inserted Error");
            else
                MessageBox.Show("Data Inserted");
            c.Close();
        }

        private void button2_Click(object sender, EventArgs e)
        {
            c.Open();
            cmd = new SqlCommand("select * from emp", c);
            dr = cmd.ExecuteReader();
            listBox1.Items.Clear();
            listBox2.Items.Clear();
            listBox3.Items.Clear();
            while (dr.Read())
            {
                listBox1.Items.Add(dr["eid"].ToString());
                listBox2.Items.Add(dr[1].ToString());
                listBox3.Items.Add(dr[2].ToString());
            }
            c.Close();
        }

        private void button3_Click(object sender, EventArgs e)
        {
            c.Open();
            cmd = new SqlCommand("select count(ename) from emp", c);
            Object o = cmd.ExecuteScalar();
            MessageBox.Show("Total employee=" + o.ToString());
            c.Close();
        }

        private void Form1_Load(object sender, EventArgs e)
        {

        }

        private void button4_Click(object sender, EventArgs e)
        {
            c.Open();
            cmd = new SqlCommand("sam", c);
            cmd.CommandType = CommandType.StoredProcedure;

            dr = cmd.ExecuteReader();
            listBox1.Items.Clear();
            listBox2.Items.Clear();
            listBox3.Items.Clear();
            while (dr.Read())
            {
                listBox1.Items.Add(dr[0].ToString());
                listBox2.Items.Add(dr[1].ToString());
                listBox3.Items.Add(dr[2].ToString());
            }
            c.Close();
        }

        private void button5_Click(object sender, EventArgs e)
        {
            c.Open();
            cmd = new SqlCommand("insr", c);
            cmd.CommandType = CommandType.StoredProcedure;
            cmd.Parameters.Add(new SqlParameter("@eno",textBox1.Text));
            cmd.Parameters.Add(new SqlParameter("@ename", textBox2.Text));
            cmd.Parameters.Add(new SqlParameter("@sal", textBox3.Text));
            dr = cmd.ExecuteReader();
            
            listBox1.Items.Clear();
            listBox2.Items.Clear();
            listBox3.Items.Clear();
            while (dr.Read())
            {
                listBox1.Items.Add(dr[0].ToString());
                listBox2.Items.Add(dr[1].ToString());
                listBox3.Items.Add(dr[2].ToString());
            }
            c.Close();

        }

        private void button6_Click(object sender, EventArgs e)
        {
            c.Open();
            cmd = new SqlCommand("spemp", c);
            cmd.CommandType = CommandType.StoredProcedure;
            cmd.Parameters.Add(new SqlParameter("@eno", textBox1.Text));
            cmd.Parameters.Add(new SqlParameter("@ename", textBox2.Text));
            cmd.Parameters.Add(new SqlParameter("@sal", textBox3.Text));
            dr = cmd.ExecuteReader();

            listBox1.Items.Clear();
            listBox2.Items.Clear();
            listBox3.Items.Clear();
            while (dr.Read())
            {
                listBox1.Items.Add(dr[0].ToString());
                listBox2.Items.Add(dr[1].ToString());
                listBox3.Items.Add(dr[2].ToString());
            }
            c.Close();
        }

        private void button7_Click(object sender, EventArgs e)
        {
            string s = listBox1.SelectedItem.ToString();
            c.Open();
            cmd = new SqlCommand("delete from emp where eid="+s,c);

            int a = cmd.ExecuteNonQuery();
            if (a == 0)
                MessageBox.Show("Data Deleted Error");
            else
                MessageBox.Show("Data Deleted ");
            c.Close();


        }

        



    }
}

File: ADO/ado_vb/ado_vb/Form1.vb
Imports System.Data
Imports System.Data.SqlClient

Public Class Form1
    Dim c As SqlConnection
    Dim cmd As SqlCommand
    Dim dr As SqlDataReader

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        Dim i As Integer
        c.Open()
        cmd = New SqlCommand("insert into emp values(" + TextBox1.Text + ",'" + TextBox2.Text + "'," + TextBox3.Text + ")", c)
        i = cmd.ExecuteNonQuery()
        If i > 0 Then
            MsgBox("Data Inserted")
        Else
            MsgBox("Data Inserted Error")
        End If
        c.Close()
    End Sub

    Private Sub Form1_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        c = New SqlConnection("Data Source=SUNETHRA\ARASU;Initial Catalog=ado;Integrated Security=True")

    End Sub

    Private Sub Button4_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button4.Click
        Dim i As Integer

        c.Open()
        cmd = New SqlCommand("delete from emp where eid=" + TextBox1.Text, c)
        i = cmd.ExecuteNonQuery()
        If i > 0 Then
            MsgBox("Data Deleted")
        Else
            MsgBox("Data Deleted Error")
        End If
        c.Close()

    End Sub

    Private Sub Button2_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button2.Click
        Dim i As Integer

        c.Open()
        cmd = New SqlCommand("update emp ename='" + TextBox2.Text + "', salary=" + TextBox3.Text + "where eid=" + TextBox1.Text, c)
        i = cmd.ExecuteNonQuery()
        If i > 0 Then
            MsgBox("Data Inserted")
        Else
            MsgBox("Data Inserted Error")
        End If
        c.Close()

    End Sub

    Private Sub Button3_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button3.Click

        ListBox1.Items.Clear()
        ListBox2.Items.Clear()
        ListBox3.Items.Clear()
        c.Open()
        cmd = New SqlCommand("select * from emp", c)

        dr = cmd.ExecuteReader()
        While dr.Read()
            ListBox1.Items.Add(dr(0))
            ListBox2.Items.Add(dr("ename"))
            ListBox3.Items.Add(dr(2))

        End While
        c.Close()
    End Sub


    Private Sub Button5_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button5.Click
        Dim s, v As String
        v = ListBox1.SelectedItem.ToString()

        s = ListBox1.SelectedIndex.ToString()
        ListBox1.Items.RemoveAt(s)
        ListBox2.Items.RemoveAt(s)
        ListBox3.Items.RemoveAt(s)

        Dim i As Integer
        c.Open()
        cmd = New SqlCommand("delete from emp where eid=" + v, c)
        i = cmd.ExecuteNonQuery()
        If i > 0 Then
            MsgBox("Data Deleted")
        Else
            MsgBox("Data Deleted Error")
        End If
        c.Close()
    End Sub

    Private Sub Button6_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button6.Click
        Dim o As Object

        c.Open()
        cmd = New SqlCommand("select count(eid) from emp", c)
        o = cmd.ExecuteScalar()
        MsgBox("No of data is =" & o.ToString())
        c.Close()

    End Sub
End Class
